packages/common/src/repo/repo.ts

```diff
@@ -76,8 +80,8 @@ export class Repo {
   }
 
   static async load(blockstore: IpldStore, cid: CID, authStore?: AuthStore) {
-    const commit = await blockstore.get(cid, schema.commit)
-    const root = await blockstore.get(commit.root, schema.repoRoot)
+    const commit = await blockstore.get(cid, def.commit)
```

Comment:
changed all previously used `schema`s (to describe `zod` object shapes) to `def` so we don't overload the term "schema"